### ISSUE TITLE
[npm]: downloads/day fails again

### ISSUE THREAD

**Issue by olstenlarck** [amio/badgen-service#38]
Pretty strange. Now it wants to be 3 days before ... ;d wat the heck. I totally should see how Shields handles this.

**Comment by olstenlarck** [comment]
How that's possible. That API is hilarious.

https://api.npmjs.org/downloads/point/2018-07-21/react - ok
https://api.npmjs.org/downloads/point/2018-07-22/react - ok
https://api.npmjs.org/downloads/point/2018-07-23/react - zero downloads?! :scream: 
https://api.npmjs.org/downloads/point/2018-07-24/react - ok
https://api.npmjs.org/downloads/point/2018-07-25/react - zero, okey acceptable

Huh. Shields does not have badge for `dowloads/day`? Interesting.

**Comment by amio** [comment]
Seems the api are not so stable, their data count should be correct, here's what I got for now:
https://api.npmjs.org/downloads/point/2018-07-23/react
```
{"downloads":491560,"start":"2018-07-23","end":"2018-07-23","package":"react"}
```

**Comment by olstenlarck** [comment]
> Seems the api are not so stable

For sure. Let's just remove the dowloads/day? It works i most of the cases, but when user see `0` won't be cool for his customers ;d Always will have such issues opened here because of that..

**Comment by amio** [comment]
Agree, remove it from index, leave the feature available for future evaluation. Some test cases here:

![](https://badgen.net/npm/dd/micromatch)
![](https://badgen.net/npm/dd/babel)
![](https://badgen.net/npm/dd/react)
![](https://badgen.net/npm/dd/webpack)

**Comment by olstenlarck** [comment]
we can close this?
